neard/src/config.rs

```diff
@@ -589,15 +590,31 @@ pub fn init_configs(
         .unwrap_or_else(random_chain_id);
     match chain_id.as_ref() {
         "mainnet" => {
-            // TODO:
-            unimplemented!();
+            if test_seed.is_some() {
+                panic!("Test seed is not supported for MainNet");
+            }
+            let mut config = Config::default();
+            config.telemetry.endpoints.push(MAINNET_TELEMETRY_URL.to_string());
+            config.write_to_file(&dir.join(CONFIG_FILENAME));
+
+            let genesis: Genesis = serde_json::from_str(
```

Comment:
I think we should do what we did for testnet and not store the state in the repository as it has proven to be suboptimal. Rather we should take genesis file as an argument to the function.

Reply:
For phase 0 genesis it should be fine.

Reply:
Agree, it's fine for launching Phase 0, but will need to upgrade later.

I do think we need to fix current `near init` crashing without specifying the genesis records file though.